src/main/java/nl/praegus/fitnesse/symbols/MavenProjectVersions/VersionNumber.java

```diff
@@ -0,0 +1,32 @@
+package nl.praegus.fitnesse.symbols.MavenProjectVersions;
+
+public class VersionNumber{
+    public final int[] versionNumberArray;
```

Comment:
Variabelen mogen (in principe) niet public zijn!

Reply:
Variabele private gemaakt